packages/jupyterlab-manager/src/index.ts

```diff
@@ -2,9 +2,23 @@
 // Distributed under the terms of the Modified BSD License.
 
 import WidgetManagerProvider from './plugin';
+import * as output from './output';
 
 export default WidgetManagerProvider;
 
 export {
   INBWidgetExtension
 } from './plugin';
+
+export {
+  WidgetManager
+} from './manager';
+
+export {
+  WidgetRenderer
+} from './renderer';
+
+export {
+  output
+};
```

Comment:
Can this just be `export {output} from './output';`, instead of having the `import *` above?

Reply:
never mind, I see that you're just exporting a namespace.